### DISCUSSION TITLE
Paypal?

### DISCUSSION THREAD

**Discussion by JimmysMaths** [samolego/Canta#119] [General]
can i send you some money to thank you for this?

**Comment by samolego** [comment]
Hey! Thanks, but you might consider donating to [UAD](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/) first, since Canta relies on it for all the badge info :).

My [donation link](https://www.paypal.com/donate/?hosted_button_id=HPUP9R6VH7FTN).

**Reply by JimmysMaths** [reply]
The donation link doesn't work. ![Screenshot_20241121-063922_Mull.jpg](https://github.com/user-attachments/assets/999c6574-b2e4-4885-8e3d-6e6b36b85ed7)

**Reply by samolego** [reply]
Humm, works for me even in private window ... perhaps try a different browser?

**Comment by samolego** [comment]
Tysm <3
